signac/contrib/job.py

```diff
@@ -167,7 +167,11 @@ def ws(self):
         return self.workspace()
 
     def reset_statepoint(self, new_statepoint):
-        """Reset the state point of this job.
+        """Overwrite the state point of this job while preserving job data.
+
+        Change the job id if necessary.
+        For more, see
```

Comment:
I would use the following sentence to more clearly indicate the behavior. Also I would add a blank line so that this sentence is in its own paragraph and stands out from the following sentence.
```suggestion
        This method will change the job id if the state point has been altered.
        
        For more information, see
```